library/std/src/io/buffered/bufreader.rs

```diff
@@ -99,7 +99,10 @@ impl<R: Read> BufReader<R> {
 impl<R: Read + ?Sized> BufReader<R> {
     /// Attempt to look ahead `n` bytes.
     ///
-    /// `n` must be less than `capacity`.
+    /// `n` must be less than or equal to `capacity`.
+    ///
+    /// the returned slice may be less than `n` bytes long if
```

Comment:
This should be capitalized, shouldn't it?

Reply:
yeah, nice catch.  i plan on doing another pass at the documentation once the finer details of the behavior are worked out, possibly earlier.

however, this is a very minor typo, and i don't think it's worth blocking such a critical fix on such a minor typo.